VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -37,6 +37,15 @@
             android:textSize="@dimen/edx_small"
             tools:text="Thread body" />
 
+        <TextView
+            android:id="@+id/discussion_responses_thread_row_visibility_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
```

Comment:
@aleffert, any comments on the iOS implementation? Is the margin correct, or should it be removed?